ArduCopter/RC_Channel.cpp

```diff
@@ -203,7 +203,7 @@ bool RC_Channel_Copter::do_aux_function(const aux_func_t ch_option, const AuxSwi
             if ((ch_flag == AuxSwitchPos::HIGH) &&
                 (copter.flightmode->allows_save_trim()) &&
                 (copter.channel_throttle->get_control_in() == 0)) {
-                copter.save_trim();
+                copter.g2.rc_channels.save_trim();
```

Comment:
is it a bit weird this feature goes all the way down into the rc_channels object instead of calling the copter feature directly?

Reply:
There's nothing in these functions which make them Copter-specific - if we were insane we could let quadplane use the same functions in qstab!  Same with Sub if they used RC.  Less so with Rover :-)

Moving it up would mean save_trim was available on `rc()` which would make this a little less odd (and it would be in the parent class...)

Oh, and I forgot about Blimp, which could share the same functions.

Reply:
ok, txs for the explanation.. I've thought about it a bit more and I think you're right that it's fine here.